dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -9,6 +9,9 @@ class AdminReportsController < ApplicationController
   before_action :set_script
   include LevelSourceHintsHelper
 
+  def directory
```

Comment:
there's GOTTA be a rails pattern for direct template rendering without needing to define an empty view method